tianshou/policy/modelfree/dqn.py

```diff
@@ -26,6 +26,7 @@ class DQNPolicy(BasePolicy):
     :param bool reward_normalization: normalize the reward to Normal(0, 1).
         Default to False.
     :param bool is_double: use double dqn. Default to True.
+    :param bool clip_delta: use clipping. Default to True.
```

Comment:
`clip_delta` is quite confusing at the first glance...

Reply:
Should we change to just clipping then? The idea is that what is effectively getting clipped are the gradients, not the values in the forward run. I am not sure what the best name would be.

Reply:
So at least you should state these in the docstring right?

Reply:
> So at least you should state these in the docstring right?

Yes, that's a fair point, I will take care of it.